torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -259,14 +259,6 @@ RegisterOperators reg({
           return 0;
         };
       }),
-    Operator(
```

Comment:
Can we really keep deleting operators like that? Won't this break existing checkpoints??

Reply:
Probably, should we instead add this back into a separate list of legacy/deprecated ops?

Reply:
I think it should work fine if we translate calls to prim::NoneGenerator into prim::None when we load the model in import_methods. But it should be tested to see that an old network works.